src/gpodder/util.py

```diff
@@ -1480,14 +1480,32 @@ def gui_open(filename):
     """
     try:
         if gpodder.ui.win32:
-            os.startfile(filename)
+            opener = None
         elif gpodder.ui.osx:
-            Popen(['open', filename], close_fds=True)
+            opener = 'open'
         else:
-            Popen(['xdg-open', filename], close_fds=True)
+            opener = 'xdg-open'
+
+        if opener is None:
```

Comment:
It would be better to leave startfile in the windows section above in case additional branches are added above that set opener=None. This section should only handle `if opener:`.

Reply:
Indead, that makes sense